recipes/vsg/all/conanfile.py

```diff
@@ -0,0 +1,118 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc_static_runtime, is_msvc
+from conan.tools.files import get, copy, rm, rmdir, collect_libs
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMakeToolchain, CMakeDeps, CMake
+import os
+
+required_conan_version = ">=1.53.0"
+
+class VsgConan(ConanFile):
+    name = "vsg"
+    description = "VulkanSceneGraph"
```

Comment:
Would it be possible to provide a more meaningful description please?

Reply:
I've modified the PR [here](https://github.com/conan-io/conan-center-index/pull/18996)